### PR TITLE
Add Trevor Rosen and Santiago Torres-Arias to TSC

### PR DESCRIPTION
**Pull request by lukehinds**
Signed-off-by: Luke Hinds <lhinds@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,8 @@ Meetings are also recorded and posted to the [sigstore YouTube channel](https://
 | Dan Lorenc | dlorenc@chainguard.dev | Chain Guard |
 | Bob Callaway | bcallaway@google.com | Google |
 | Luke Hinds (TSC Chair) | lhinds@redhat.com | Red Hat |
+| Trevor Rosen | trevrosen@github.com | GitHub |
+| Santiago Torres-Aria | santiagotorres@purdue.edu | Purdue University |
```
Comment (dlorenc) [inline README.md:46]: Torres-Arias, typo

**Review by bobcallaway — APPROVED** [review]

**Comment by dlorenc** [comment]
Congrats, and welcome to the TSC @SantiagoTorres and @trevrosen!

**Comment by dlorenc** [comment]
For context, see the TSC minutes [from today](https://docs.google.com/document/d/1yr7kib0jgmPbIM0nuNUa28_ur-Ze_pfhgCW9f78oL0I/edit).

**Comment by trevrosen** [comment]
🎉 thanks! Excited to be here! 🥳

**Comment by SantiagoTorres** [comment]
Thank you!

**Comment by dlorenc** [comment]
And finally, I plan to step down from the TSC soon. It's been an awesome term so far, but I no longer have the time required to do this job well.

Once we've transitioned these two new members on, I'll plan on stepping down myself and helping to hopefully transition on @priyawadhwa in my place!

**Pull request merged**